src/models/Move.php

```diff
@@ -113,8 +113,13 @@ public static function prepareDescr(?string $descr): array|string|null
             return preg_replace_callback('@https://\S+/(\d+)/?(#\S+)?@', static function ($m) {
                 return Html::a('HM4::' . Html::encode($m[1]), Html::encode($m[0]));
             }, $descr);
+        } else if (str_contains($descr, 'youtrack')) {
+            $urlPattern = '/(http|https)\:\/\/?youtrack.ahback.com\/issue\/([A-Z]+\-\d+)[a-zA-Z0-9\&\.\/\?\:@\-_=#]*/';
+            return preg_replace_callback($urlPattern, function ($url) {
+                return Html::a($url[2], $url[0]);
+            }, $descr);
```

Comment:
The code should not disclose the real service name in a public repo!!
YT has different URL formats that include an issue ID. Make your implementation more abstract and vague.

Simply for links with `[A-Z]+\-\d+` and run preg_replace_callback over them fo format accordingly.